libs/domain/content/src/models/content.model.ts

```diff
@@ -1,17 +1,16 @@
 import { ContentEntities } from '../services';
-import { RouteType } from '@spryker-oryx/router';
 
 export interface ContentQualifier {
   type?: string;
   id?: string;
+  query?: string;
   entities?: ContentEntities;
 }
 
-export interface Content {
-  id?: string;
-  type?: RouteType;
-  url?: string;
-  heading: string;
-  description: string;
-  content: string;
+export interface Content<T = Record<string, unknown>> {
+  fields: T & {
+    id: string;
+  };
+  id: string;
+  version: number;
```

Comment:
just wondering, what we use version for in the storefront?

Reply:
we dont' but I thought mb it will be valuable info for cms. But I can move it to the EB only?